_pkgdown.yml

```diff
@@ -124,7 +124,12 @@ reference:
 
 articles:
 
+  - title: Basic Data Wrangling
+    navbar: ~
+    contents:
+    - basic_data_wrangling
```

Comment:
If you name it "datawizard.Rmd", it will be rendered as "Get Started" vignette.

I think this vignette is the perfect candidate for that.

Reply:
I think, this vignette might not be the best fit for "Get Started" since it focuses exclusively on data wrangling functionality, and we also need to talk about data transformations. 

So we can keep the name as it is, and when we work on the JOSS paper, we can convert that paper to "Get Started" (https://github.com/easystats/easystats/issues/196).